config-example/1shardproxy.toml

```diff
@@ -0,0 +1,39 @@
+log_level = "DEBUG5"
```

Comment:
Add toml and json configs just to show that it is possible to use difference format